backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -354,9 +355,18 @@ public void draw (GLKView view, CGRect rect) {
 		}
 
 		long time = System.nanoTime();
-		deltaTime = (time - lastFrameTime) / 1000000000.0f;
+		// After pause deltaTime can have somewhat huge value that destabilizes the mean, so let's cut it off
+		if (!resume) {
+			deltaTime = (time - lastFrameTime) / 1000000000.0f;
+		} else {
+			deltaTime = 0;
+		}
 		lastFrameTime = time;
 
+		if (resume) {
+			resume = false;
+		}

```

Comment:
Minor thing but we can probably remove this block and set resume to false on the previous conditional that checks resume value.

Reply:
You're right, it's kind of redundant.

Reply:
thanks for the changes, I'll test it as soon as I can